deepfence_bootstrapper/controls/controls.go

```diff
@@ -81,11 +82,16 @@ func SetAgentControls() {
 			if err != nil {
 				return err
 			}
-			err = scanner.RunComplianceScan()
-			if err != nil {
-				log.Error().Msgf("Error from scan: %+v", err)
-				return err
-			}
+
+			log.Info().Msg("StartComplianceScan Starting")
+			//We need to run this in a goroutine else it will block the
+			//fetch and execution of controls
+			go func() {
+				err := scanner.RunComplianceScan()
+				if err != nil {
+					log.Error().Msgf("Error from RunComplianceScan: %+v", err)
+				}
+			}()
```

Comment:
Good find - we need to check all the other scans

Reply:
👍